src/osemosys_global/OPG_powerplant_data.py

```diff
@@ -151,77 +98,34 @@ def main(model_start_year=2015, model_end_year=2050, region_name='GLOBAL'):
                         'SA-BRA-J2',
                         'SA-BRA-J3',
                         'SA-SUR',]
-    nodes_extra_df['node'] = nodes_extra_list
-
-    df_gen_agg_node = df_gen_agg_node.append(nodes_extra_df,
-                                            ignore_index=True,
-                                            sort='False').fillna(0).sort_values(by='node').set_index('node').round(2)
-    #df_gen_agg_node.to_csv(r'output/test_output_2.csv')
-
-
-    # Add region and country code columns
-    df_gen_2['region_code'] = df_gen_2['node'].str[:2]
-    df_gen_2['country_code'] = df_gen_2['node'].str[3:]
 
+    # TODO The following aggregated capacity code doesn't seem to be used
 
-    # ### Add operational life column
-    op_life_dict = dict(zip(list(df_op_life['tech']),
-                            list(df_op_life['years'])))
+    # # Create table with aggregated capacity
```

Comment:
@abhishek0208 - this aggregated capacity code doesn't seem to be used (the csv file writing was commented out). Should it be kept, or can it be removed?

Reply:
@willu47 Oh yes, that was to cross-check aggregate capacities against @maartenbrinkerink summary statistics. It isn't used as part of the OSeMOSYS global data package itself so can be removed.